pkg/network/usm/procnet/parser.go

```diff
@@ -0,0 +1,223 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
```

Comment:
All files in the directory are linux only - I think that breaks compilation for windows and mac

Reply:
the windows and macOS builds seem to be working (`macos_lint` seems broken but apparently for unrelated reasons`)
I believe it's OK for all files in the package to have the `linux_bpf` tag as long as they're not important by the macOS or windows builds